test-requirements.txt

```diff
@@ -6,4 +6,5 @@ pytest-timeout
 pytest-asyncio
 pytest-httpx
 flake8-docstrings
-flake8
\ No newline at end of file
+flake8
+asynctest
```

Comment:
Agreed, I hadn't done tests like this in python before so I was just following some suggestions from google. Any chance you know of another library? Otherwise I'll find one.

Reply:
It might work simply replacing `asynctest.patch` by `unittest.mock.patch`

Reply:
Couple other small tweaks needed to use that, but yup that did it. PR updated.

Reply:
The tests seem to work, but there is a pylint error now

Reply:
Can you run it now? For some reason I can't run pylint locally against the tests directory so I'm unfortunately resolving the issues manually:

```
************* Module tests
tests\__init__.py:1:0: F0010: error while code parsing: Unable to load file tests\__init__.py:
[Errno 2] No such file or directory: 'tests\\__init__.py' (parse-error)
```